src/docs/Core-2.0/Api/CapabilityApi.md

```diff
@@ -0,0 +1,66 @@
+CapabilityApi
+=============
+
+classname: \Zikula\ExtensionsModule\Api\CapabilityApi
+
+service id="zikula_extensions_module.api.capability"
+
+This class defines and tests for the capabilities of an extension. This class replaces similar functionality
+from ModUtil::* methods.
+
+The class makes the following methods available:
+
+    - getExtensionsCapableOf($capability)
+    - isCapable($extensionName, $requestedCapability)
+    - getCapabilitiesOf($extensionName)
+
+The class is fully tested.
+
+CapabilityApiInterface
+----------------------
+
+classname: \Zikula\ExtensionsModule\Api\ApiInterface\CapabilityApiInterface
```

Comment:
ah - I was just preparing for the possibility of other interfaces. I would prefer `\Zikula\ExtensionsModule\Api\Interface\CapabilityApiInterface` but `Interface` is a reserved word.